lib/bibliothecary/parsers/maven.rb

```diff
@@ -274,7 +274,7 @@ def self.parse_maven_resolved(file_contents, options: {})
       def self.parse_maven_tree_items_with_depths(file_contents)
         file_contents
           .gsub(ANSI_MATCHER, "")
-          .gsub(/\r\n?/, "\n")
+          .encode(universal_newline: true)
```

Comment:
Switched Maven to use this too, docs here: https://github.com/ruby/ruby/blob/master/doc/encodings.rdoc#encoding-options-